firmware/hw_layer/drivers/gpio/tle8888.c

```diff
@@ -364,7 +421,9 @@ static THD_FUNCTION(tle8888_driver_thread, p) {
 				/* set state to TLE8888_FAILED? */
 			}
 
-			ret = tle8888_update_status(chip);
+			/* this is expensive call, will do a lot of spi transfers...
+			 * make it conditional? */
+			ret = tle8888_update_status_and_diag(chip);
```

Comment:
agree - we probably don't want to do this every time.  Perhaps only if it's been 50/100/500ms since the last read? (if we do something like PWM-over-spi, this is going to be bad)